client/disputes/new-evidence/index.tsx

```diff
@@ -0,0 +1,84 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import { Button } from 'wcpay/components/wp-components-wrapped';
+import '../style.scss';
+import { Stepper, StepperPanel, useStepperContext } from 'components/stepper';
```

Comment:
```suggestion
import { Button } from 'wcpay/components/wp-components-wrapped';
import '../style.scss';
import { Stepper, StepperPanel, useStepperContext } from 'wcpay/components/stepper';
```

( ⛏️ ) To maintain consistency with the import location.